FanartHandler/FanartSelectedOther.cs

```diff
@@ -151,9 +151,26 @@ public void RefreshGenericSelectedProperties(Utils.SelectedType property, ref st
             }
             AddSelectedGenreProperty(SelectedGenre, SelectedItem, property);
 
-            if (isVideo)
+            // Trakt remove ClearArt for pages

```

Comment:
It would be more correct, in my opinion, not to remove properties here, but to immediately sift them out in the method:
https://github.com/andrewjswan/mediaportal-fanart-handler/blob/b7e3f4547b7a2a9eb4e220bfa658fc6240b10237/FanartHandler/Utils.cs#L2409
That is, transfer the logic from this 
https://github.com/andrewjswan/mediaportal-fanart-handler/blob/b7e3f4547b7a2a9eb4e220bfa658fc6240b10237/FanartHandler/FanartSelectedOther.cs#L154-L174
method to the above one (**GetSelectedItem**).

Reply:
> I do not follow what you mean

Change the **GetSelectedItem** function that if this is `Trakt`, and the current element `isFolder`, then return an empty value, then in all parts where this function is called we will always receive an empty value for the Folder in Trakt, and the properties will **not be filled**.

Reply:
It's not just Trakt, it's for everything that needs the clearart string value to not clear. I thought we were not going to use idFolder because clearart can be on a collection.